Platforms/QemuQ35Pkg/ConfigKnobs/ConfigDataGfx/ConfigDataGfx.c

```diff
@@ -64,16 +64,23 @@ ApplyGfxConfigToPolicy (
   GfxEnablePort0   = *(BOOLEAN *)ConfigBuffer;
   GfxSiliconPolicy = AllocateCopyPool (sizeof (DefaultQemuGfxPolicy), DefaultQemuGfxPolicy);
 
+  if (GfxSiliconPolicy == NULL) {
+    DEBUG ((DEBUG_ERROR, "Failed to allocate Policy structure\n"));
+    goto Exit;
+  }  // Mu

```

Comment:
```suggestion
  }
```

Reply:
As mentioned in Oliver's comment, the "Mu" comments can be dropped.

Reply:
Makes sense.  Updated w/ recommendation.